packages/orchestration/src/exos/portfolio-holder-kit.js

```diff
@@ -0,0 +1,200 @@
+        harden({
+          accounts,
+          publicTopics,
+        })
+      );
+    },
+    {
+      invitationMakers: {
+        /**
+         * @param {string} chainName key where the account is stored
+         * @param {string} action invitation maker name, e.g. 'Delegate'
+         * @param {unknown[]} invitationArgs
+         */
+        Action(chainName, action, invitationArgs) {
+          const { accounts } = this.state;
+          accounts.has(chainName) || Fail`no account found for ${chainName}`;
+          const account = accounts.get(chainName);
+          return watch(
+            E(account).asContinuingOffer(),
+            this.facets.getInvitationMakersWatcher,
+            { action, invitationArgs },
+          );
+        },
+      },
+      holder: {
+        // XXX type. like `OrchestrationAccountI['asContinuingOffer']` but
+        // returns a Vow<Invitation> instead of a Promise<Invitation> as the
+        // accounts are remote
+        asContinuingOffer() {
+          const { invitationMakers } = this.facets;
+          const { publicTopics } = this.state;
+          return harden({
+            publicSubscribers: fromEntries(publicTopics.entries()),
+            invitationMakers,
+          });
+        },
+        /** @returns {ResolvedTopicsRecord} */
+        getPublicTopics() {
+          const { publicTopics } = this.state;
+          return harden(fromEntries(publicTopics.entries()));
+        },
+        /**
+         * @param {string} chainName key where the account is stored
+         * @param {OrchestrationAccount<any>} account
+         * @param {ResolvedPublicTopic<unknown>} publicTopic
+         */
+        addAccount(chainName, account, publicTopic) {
+          if (this.state.accounts.has(chainName)) {
+            throw Fail`account already exists for ${chainName}`;
+          }
+          zone.isStorable(account) ||
+            Fail`account for ${chainName} must be storable`;
+          zone.isStorable(publicTopic) ||
+            Fail`publicTopic for ${chainName} must be storable`;
+
+          this.state.publicTopics.init(chainName, publicTopic);
+          this.state.accounts.init(chainName, account);
+        },
+        /**
+         * @param {string} chainName key where the account is stored
+         */
+        getAccount(chainName) {
+          return this.state.accounts.get(chainName);
+        },
+      },
+      getInvitationMakersWatcher: {
+        /**
+         * @param {ResolvedContinuingOfferResult} result
+         * @param {{ action: string; invitationArgs: unknown[] }} ctx
+         * @returns {Promise<Invitation>}
+         */
+        onFulfilled({ invitationMakers }, { action, invitationArgs }) {
+          return E(invitationMakers)[action](...invitationArgs);
+        },
+      },
+    },
+  );
+};
+
+/** @typedef {ReturnType<typeof preparePortfolioHolderKit>} MakePortfolioHolderKit */
+/** @typedef {ReturnType<MakePortfolioHolderKit>} PortfolioHolderKit */
```

Comment:
typedef will export the types. But because the kit preparer isn't export these aren't to be used outside the module.
```suggestion
```